### PR TITLE
For production

### PR DESCRIPTION
**Pull request by Salman9000**

### CODE REVIEW

File: Queensman-Team-deploy-ready/src/Notification/component/NotificationList.js [hunk 1]
```diff
@@ -110,22 +119,36 @@ export default function NotificationList({
   };
   if (!item.isRead) {
     return (
+      <>
+      <Pressable
+        onLongPress={() => {
+          console.log("pre2ss")
```
Comment (mustafa-hanif) [inline Queensman-Team-deploy-ready/src/Notification/component/NotificationList.js:125]: remove logs

File: admin-panel/src/views/clients/TableAdvSearch.js [hunk 2]
```diff
@@ -411,7 +425,12 @@ const DataTableAdvSearch = () => {
     let month = new Date().getMonth() + 1
     let day = new Date().getDate()
     const propertyid = row.property_owneds[0]?.property?.id ? row.property_owneds[0]?.property?.id : row.leases[0]?.property?.id
-
+    const notesArray =  [
```
Comment (mustafa-hanif) [inline admin-panel/src/views/clients/TableAdvSearch.js:428]: What is this hard coded stuff

**Review by mustafa-hanif — COMMENTED** [review]
Why did you add react-offline package?

File: admin-panel/src/views/clients/TableAdvSearch.js (hunk 2 above)
Comment (Salman9000) [reply admin-panel/src/views/clients/TableAdvSearch.js:428]: These are labels for upload plan scheduled services

**Comment by Salman9000** [comment]
> Why did you add react-offline package?

Admin panel acts strange when there is discrepancy in internet connection. I added this package to let user know that internet is not working

File: Queensman-Team-deploy-ready/src/Notification/component/NotificationList.js (hunk 1 above)
Comment (Salman9000) [reply Queensman-Team-deploy-ready/src/Notification/component/NotificationList.js:125]: all logs or this one only

**Pull request merged**